src/main/features/first-contact/views/claim-summary.njk

```diff
@@ -37,6 +33,9 @@
           <p>{{ t('Find out about how we use and store your <a href="{{ link }}" target="_self">personal information</a>.', { link: AppPaths.privacyPolicyPage.uri }) | safe }}</p>
         </div>
       </details>
+
+      <p class="bold-small">{{ t('Download the claim') }}</p>
+      <p><span class="block"><a href="{{ FirstContactPaths.receiptReceiver.uri }}">{{ t('Download claim') }}</a> ({{ t('PDF') }})</span></p>
```

Comment:
do we really need this span? just leave it in the p and add the class there if needed? although I doubt you will need the class

Reply:
Done